easy_thumbnails/processors.py

```diff
@@ -35,6 +35,27 @@ def _points_table():
             yield j
 
 
+def _call_pil_method(im, method, *args, **kwargs):
```

Comment:
I'd suggest a wrapper class that looked something like:
```py
from functools import partial

class FrameAware:
    def __new__(cls, im):
        if getattr(im, "n_frames", 1) > 1:
            return super().__new__(cls)
        return im

    def __init__(self, im):
        self.im = im

    def apply_to_frames(self, method, *args, **kwargs):
        results = []
        for i in range(self.im.n_frames):
            self.im.seek(i)
            result = method(*args, **kwargs)
            self.im.paste(result)
        return self.im

    def __getattr__(self, key):
        method = getattr(self.im, key)
        return partial(self.apply_to_frames, method)
```
Now you can do:
```py
im = FrameAware(im).convert(new_mode)
```

Reply:
Ah, looks nice. Will try it.